challenges/trees/__tests__/trees.test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const testModule = require('../trees');
+
+
+// Tree Tests
+
+
+
+describe('Tree Tests', () => {
+  it('Can successfully instantiate an empty tree', () => {
+
+
+  });
+
+
+
+  it('Can successfully instantiate a tree with a single root node', () => {
+
+
+  });
+
+
+
+
+  it('Can successfully add a left child and right child to a single root node', () => {
+
+
+  });
+
+
+
+
+  it('Can successfully return a collection from a preorder traversal', () => {
+
+
+  });
+
+
+
+  it('Can successfully return a collection from an inorder traversal', () => {
+
+
+  });
+
+
+
+  it('Can successfully return a collection from a postorder traversal', () => {
+
+
+  });
+
+
+
+
+});
+
+
```

Comment:
Code is looking great, once you get these tests filled out and that README updated then I will regrade for you!